package.json

```
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@epam/ai-dial-shared": "^0.39.1",
-    "@epam/ai-dial-ui-kit": "0.5.0-rc.40",
+    "@epam/ai-dial-ui-kit": "0.5.0-rc.44",
```

latest version
```suggestion
    "@epam/ai-dial-ui-kit": "0.5.0-rc.45",
```
did you execute `npm ddp`? (deduplication dependencies in lock-file)

---

ui-kit updated. 
npm ddp - another PR. For security checks